Contest_API.md

```diff
@@ -1381,7 +1485,7 @@ The request must fail with a 4xx error code if any of the following happens:
 The response will contain a `Location` header pointing to the newly created submission
 and the response body will contain the initial state of the submission.
 
-Performing a `POST` or `PUT` by any roles other than `admin` and `team` is not supported.
+Performing a `POST` or `PUT` is not supported except when the capabilities are available.
```

Comment:
Do we actually need to state this here, as it's already said at the top of this section?
If so, I think it would be better to write
```suggestion
Performing a `POST` or `PUT` is not supported when these capabilities are not available.
```
to remove the ambiguity that you might need more or all of these capabilities.

Reply:
The rewrite is clearer. :+1:

I do think it makes sense to state this here though.